### PR TITLE
Improve install script kitchen tests

### PR DESCRIPTION
**Pull request by kbogtob**
### What does this PR do?

Improve the install script kitchen tests and generalise the install script parameters.

### Motivation

Making the kitchen tests more testable and make them use environment variables.

### CODE REVIEW

**Review by KSerrania — DISMISSED** [review]
LGTM, left a small comment on the env var names.

Install script kitchen tests are okay, let's monitor potential install script errors since the new version of the install script will be live as soon as this PR gets merged.

Should the new environment variables for the install script be documented (and if yes, where should they)?

File: cmd/agent/install_script.sh [hunk 1]
```diff
@@ -99,6 +111,18 @@ if [ -n "$DD_AGENT_DIST_CHANNEL" ]; then
...
+  yum_version_path=$YUM_VERSION_PATH
+else
+  yum_version_path="${dd_agent_dist_channel}/${dd_agent_major_version}"
+fi
+
+if [ -n "$APT_REPO_VERSION" ]; then
+  apt_repo_version=$APT_REPO_VERSION
+else
+  apt_repo_version="${dd_agent_dist_channel} ${dd_agent_major_version}"
+fi
```
[the remark this answers is not in the document]
Comment (kbogtob) [reply cmd/agent/install_script.sh:124]: I just thought they could be "hidden" entry points for tests. I don't see why you would not set dd_agent_dist_channel and dd_agent_major_version for yum but not for apt.

[the remark this answers is not in the document]
Comment (albertvaka) [reply cmd/agent/install_script.sh:124]: We would need to change the testing repos (eg: `repo_component_apt`) to be 6/7 to do that. Is that easy to do?

[the remark this answers is not in the document]
Comment (albertvaka) [reply cmd/agent/install_script.sh:124]: And what we set as `pipeline-2419589-a6` needs to be either `stable`/`beta` which we can't do.

[the remark this answers is not in the document]
Comment (albertvaka) [reply cmd/agent/install_script.sh:124]: I would change the variables to $TESTING_* so it's clear why they are here and that's it, the alternative seems to have too many implications.

**Review by albertvaka — APPROVED** [review]

**Pull request merged**